src/setup/cortx_deploy

```diff
@@ -58,6 +58,9 @@ function conf_get {
 DEBUG_FLAG=false
 SOLUTION_CONFIG="/etc/cortx/solution"
 CORTX_CONFSTORE="yaml:///etc/cortx/cluster.conf"
+RELEASE_INFO="yaml:///opt/seagate/cortx/RELEASE.INFO"
+DEPLOY=false
```

Comment:
Lets use DEPLOY_FLAG and UPGRADE_FLAG to be consistent for boolean values

Reply:
Done